utils/src/main/kotlin/DeclaredLicenseProcessor.kt

```diff
@@ -29,7 +29,8 @@ import com.here.ort.spdx.SpdxExpression
 object DeclaredLicenseProcessor {
     private val prefixesToRemove = listOf(
         "https://choosealicense.com/licenses/",
-        "https://opensource.org/licenses/"
+        "https://opensource.org/licenses/",
+        "https://www.gnu.org/licenses/old-licenses/"
```

Comment:
@sschuberth Please also add https://licenses.nuget.org, https://tldrlegal.com/license and https://spdx.org/licenses

Reply:
Done.

Reply:
Add http://www.opensource.org/licenses/ as well